### PR TITLE
 feat: LNURL-pay comment 

### LINKED ISSUE #2000: [Bug]: LNURL bottom-sheet opens twice
### Describe the bug

Interface bug where scanning a Lightning Network (LN) URL consistently opens the modal popup twice. It briefly appears, closes (200ms), and then reopens with LN URL Payment UI details. We should ensure it opens correctly on the first attempt.

### Screenshots / Recording

_No response_

### Operating system

iOS

### Bitkit version

_No response_

### Log output

_No response_

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

- move lnurl pay screen into SendNavigation
- lnurl pay now has dedicated confirm screen, with comment input field
- new rn-ldk with new types
- e2e tests

### Linked Issues/Tasks

closes: #2000 #1680

### Type of change

New feature

### Tests

Detox test

### Screenshot / Video

https://github.com/user-attachments/assets/0c69587c-4d75-4bbf-b396-8a29c9dac6d9

### CODE REVIEW

File: src/navigation/bottom-sheet/SendNavigation.tsx [hunk 1]
```diff
@@ -125,6 +130,10 @@ const SendNavigation = (): ReactElement => {
 		}
 	};
 
+	// if max === min sendable amount, skip the Amount screen
+	// const initialRouteName =
+	// 	pParams.minSendable === pParams.maxSendable ? 'Confirm' : 'Amount';
+
```
Comment (pwltr) [inline src/navigation/bottom-sheet/SendNavigation.tsx]: remove unused code

File: src/store/types/ui.ts [hunk 2]
```diff
@@ -21,13 +21,18 @@ export type ViewControllerParamList = {
...
+		// prettier-ignore
+		| { screen: 'LNURLConfirm'; pParams: LNURLPayParams; url: string; amount?: number; }
+		| undefined;
 	timeRangePrompt: undefined;
 	transferFailed: undefined;
 	treasureHunt: { chestId: string };
 	tagsPrompt: undefined;
 	lnurlWithdraw: { wParams: LNURLWithdrawParams };
-	lnurlPay: { pParams: LNURLPayParams };
+	// lnurlPay: { pParams: LNURLPayParams };
```
Comment (pwltr) [inline src/store/types/ui.ts]: remove unused code

File: src/screens/Wallets/Send/ReviewAndSend.tsx [hunk 3]
```diff
@@ -236,7 +236,7 @@ const ReviewAndSend = ({
 			return;
 		}
 
-		refreshWallet({ onchain: false, lightning: true }).then();
+		// refreshWallet({ onchain: false, lightning: true }).then();
```
Comment (pwltr) [inline src/screens/Wallets/Send/ReviewAndSend.tsx]: remove unused code

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Wallets/LNURLPay/Confirm.tsx [hunk 4]
```diff
@@ -1,64 +1,261 @@
...
+		}
+	}, [pin, pinForPayments, biometrics, handlePay, navigateToPin]);
+
+	const fixedAmount = pParams.minSendable === pParams.maxSendable;
 
 	return (
 		<>
 			<GradientView style={styles.container}>
 				<BottomSheetNavigationHeader
 					title={t('lnurl_p_title')}
```
Comment (pwltr) [inline src/screens/Wallets/LNURLPay/Confirm.tsx:169]: Title should be "Pay Bitcoin"

**Review by pwltr — CHANGES_REQUESTED** [review]

**Comment by limpbrains** [comment]
fixed

**Review by pwltr — APPROVED** [review]

**Comment by catch-21** [comment]
@limpbrains Is it possible to retain the input comment on navigate back to change value and returning to swip to pay screen?

**Comment by catch-21** [comment]
Verified functionally working in v130 d8f96e7d6475d2eef04ae08f9d9dd2c8894a8218 sending to Wallet of Satoshi LN address.

**Comment by catch-21** [comment]
Raised https://github.com/synonymdev/bitkit/issues/2104

**Pull request merged**